integration-libs/s4om/root/components/schedule-lines/schedule-lines.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Optional } from '@angular/core';
```

Comment:
Let's add E2E tests to check scheduline lines

Reply:
done @szczepaniak-michal